js/symb/leafletMap.js

```diff
@@ -0,0 +1,201 @@
+const DEFAULT_DRAW_OPTIONS = {
+   polyline: false,
+   control: true,
+   circlemarker: false,
+   marker: false,
+   multiDraw: false,
+   drawColor: { color: '#000', opacity: 0.85, fillOpacity: 0.55 }
+};
+
+const DEFAULT_MAP_OPTIONS = {
+   center: [43.64701, -79.39425],
```

Comment:
Any particular reason?

Reply:
FYI the map defaults to somewhere in like Azerbaijan for me. Maybe I'll find out why as I continue reading.

Reply:
$MAPPING_BOUNDARIES = '';
//Project bounding box; default map centering; (e.g. 42.3;100.5;18.0;-127)
Haven't Check though as I just found it